osuosl/climesync
# Better data display

fixes issue #87 
## Changes in this PR.


- [X] Improved data display for `get_times` in interactive mode
## Testing this PR.


1. Run `nosetests`
2. Run climesync, connect to the staging server, and log in as `admin` with the password `timesync-staging`
3. Run `gt` without filtering the times and see that all of the data is presented in nice tables

@osuosl/devs
